src/RealtimeEffectPanel.cpp

```diff
@@ -987,6 +987,8 @@ RealtimeEffectPanel::RealtimeEffectPanel(wxWindow* parent, wxWindowID id, const
    mEffectList = effectList;
 
    SetSizerAndFit(vSizer.release());
+
+   Bind(wxEVT_CHAR_HOOK, &RealtimeEffectPanel::OnCharHook, this);
```

Comment:
It seems that almost everywhere else the program handles `WXK_ESCAPE`, it's the key-down, not CHAR event.  Does this matter?  I don't know.  If it works, ok

Reply:
Key down event delivered to the window that has focus. wxDialog handles WXK_ESCAPE same way